core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -3986,7 +3986,7 @@ private HepProgram getTransitiveProgram() {
         .addRuleInstance(AggregateProjectMergeRule.INSTANCE)
         .build();
     final String sql = "select e.job,d.name\n"
-        + "from (select * from sales.emp where empno = 10) as e\n"
+        + "from (select * from sales.emp where ename = 'A') as e\n"
```

Comment:
Q：Why this change ?
A：empno is key column of sales.emp. This test targets to verify Aggregate pushing down, but with `empno=10`, the Aggregate can be regarded as redundant and can be optimized/removed. In order to keep the original testing intention, I replaced by `ename = 'A'`,  where `ename` is not a key column, thus `Aggregate` should be kept.

Reply:
From this point of view, we can also see this PR can help optimize and simplify result plans.